docker/compose.yml

```diff
@@ -0,0 +1,143 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+services:
+  minio:
+    image: bitnami/minio:latest
+    environment:
+      MINIO_ROOT_USER: minio
+      MINIO_ROOT_PASSWORD: minio_minio
+      MINIO_DEFAULT_BUCKETS: spark-bucket
+    hostname: minio
+    ports:
+      - 9000
+      - 9001:9001
+
+  mysql:
+    image: mysql:${MYSQL_VERSION}
```

Comment:
Could it use MariaDB instead ? 
`mysql` Docker image does not support Linux ARM64

Reply:
I see arm64 tag in the image page, so I think the mysql should support arm64?

[1] https://hub.docker.com/_/mysql

Reply:
The error I face is:
```
[+] Running 0/1
 ⠙ mysql Pulling                                                                                                                                                                      3.2s
no matching manifest for unknown in the manifest list entries
```

I thought it was the arch but maybe it just needs a new version! Debugging it!

Reply:
https://hub.docker.com/layers/mysql/library/mysql/8.0.28/images/sha256-8cdf596eb3fba84ae6d0ad683d330dcaf87ef8fb51e4d5dd9f1e289c5012e0f8?context=explore says just `OS/ARCH
linux/amd64`

[8.0.28-oracle](https://hub.docker.com/layers/mysql/library/mysql/8.0.28-oracle/images/sha256-39b3c2650e24da10a4cb31f589e4e4aff70f2f732283ccd05390781e19cec5e9?context=explore) supports ARM64.
Are you OK with `-oracle` ?

Reply:
Thanks for verification, `8.0.28-oracle` sounds good to me. Personally, I prefer to use MySQL instead of MariaDB since I don't have MariaDB production use case experience.